### ISSUE TITLE
Feature request: scripts as separate files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#94]
```
With everything lumped together in one .json file editing with the built in 
editor is the only option. External editing would be much easier if the scripts 
could be saved as individual files in a folder structure mimicking the one 
found in the autokey UI. Ideally both phrases and script could be treated this 
way with a sidecar .json metadata (trigger information, etc.) file for each, 
that would allow creating completely new items outside of the UI.
```

Original issue reported on code.google.com by `artis.ro...@gmail.com` on 23 Oct 2010 at 11:14

**Comment by GoogleCodeExporter** [comment]
```
This has been requested before and is a nice idea, but I'm not doing any 
enhancements for the foreseeable future.
```

Original comment by `cdekter` on 23 Oct 2010 at 11:23

**Comment by GoogleCodeExporter** [comment]
```
Implemented in version 0.80.0
```

Original comment by `cdekter` on 11 Sep 2011 at 7:03
- Changed state: **Fixed**
